uint32 xmin
uint32 ymin
uint32 xmax
uint32 ymax
string id 